src/obj.rs

```diff
@@ -656,12 +627,26 @@ impl<'a> Parser<'a> {
       min_normal_index: &mut usize,
       max_normal_index: &mut usize
       ) -> Result<Object, ParseError> {
-    let name = try!(self.parse_object_name());
-    try!(self.one_or_more_newlines());
+    let name = try!(self.parse_object_name()).unwrap_or(String::new());
```

Comment:
Well, I don’t agree with that. If a name is present, it cannot be an empty `String` – i.e. if you have `o\n`, that’s considered an error IIRC. Having `Option<String>` makes it ambiguous which one represents the empty name (is it either `None` or `Some(String::new())`?).